classify/clipclassifier.py

```diff
@@ -292,32 +293,41 @@ def process_file(self, filename, **kwargs):
             logging.info(" - [{}/{}] prediction: {}".format(i + 1, len(tracker.tracks), description))
 
         if self.previewer:
-            print("Exporting preview... to {}".format(mpeg_filename))
+            logging.info("Exporting preview... to {}", mpeg_filename)
             prediction_string = ""
             for label, score in self.get_clip_prediction():
                 if score > 0.5:
                     prediction_string = prediction_string + " {} {:.1f}".format(label, score * 10)
             self.previewer.export_clip_preview(mpeg_filename, tracker, self.track_prediction)
 
+
+        self.save_metadata(filename, meta_filename, tracker)
+
+        ms_per_frame = (time.time() - start) * 1000 / max(1, len(tracker.frame_buffer.thermal))
```

Comment:
put this inside the `if self.verbose`. There's no need to calculate it if it isn't used.